1.9.0

```diff
@@ -258,6 +258,40 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName":"mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-59235",
+          "description": "Delete AllActiveCampaigns List View",
+          "actionType": "deleteListView",
+          "optionsMap": {
+            "fullName": "Campaign.AllActiveCampaigns",
+            "filterScope": "Everything",
+            "operation": "delete"
```

Comment:
mdTaskListView line 38 is trying to find a 'filters' variable which probably isn't needed when you delete a list view. So skip or work around that logic in the task class.

Reply:
Edited task class to check for null instead of empty string